jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java

```diff
@@ -160,6 +160,13 @@ public void callback(DataSet dataSet) {
         };
     }
 
+    private boolean hasInitiator(String variableName, String variableValue, ProcessInstanceSummary pis) {
```

Comment:
Since you only need the value of ```initiator``` filed, maybe there's no need to pass the whole ```ProcessInstanceSummary``` object to the method, but rather you immediately pass the ```String``` i.e. ```pis.getInitiator()```.